atomic_reactor/cli/parser.py

```diff
@@ -179,6 +179,12 @@ def _add_common_task_args(task_parser: argparse.ArgumentParser) -> None:
         default=None,
         help="file to write task result",
     )
+    task_parser.add_argument(
+        "--platforms-result",
+        metavar="FILE",
+        default=None,
+        help="file to write final platforms result",
+    )
```

Comment:
Could we use `--task-result` instead of adding a new param? (We would need to update BinaryPrebuildTask.execute):
```python
    def execute(self, init_build_dirs: Optional[bool] = False) -> Dict[str, Any]:
        super().execute(init_build_dirs)
        return {
            "final_platforms": util.get_platforms(self.workflow_data)
        }
```

Reply:
no !  task-result is also for future use, I don't want to use it randomly

Reply:
tekton is about new params .....

Reply:
it would be using it for something else, task result vs plugin result

and overloading results can create only additional logic when parsing them in the future

Reply:
The prebuild task is still a task, why should it matter that it's implemented as plugins?

What do you mean additional parsing logic? Any `--task-result` of a successful task has to be JSON, so we'll be parsing JSON no matter what

Reply:
using it for different purposes, like in this case for very specific platforms result

Reply:
Hmm, alright, I don't care _that_ much if it's a new param or not. But maybe the param should only be valid for the prebuild task? (In the parser, in a new PrebuildTaskParams class etc.). Since the param does nothing in any of the other tasks

Reply:
ok param is valid only for prebuild task